File: market-app/src/main/java/com/seminav/marketapp/external/messages/UserDto.java
package com.seminav.marketapp.external.messages;

public record UserDto(
        String id,
        String username,
        String name,
        String role,
        String surname,
        String patronymic,
        String email,
        String phoneNumber,
        String tgUsername,
        String address,
        String password,
        String salt
) {}

File: bookingwm-app/src/main/kotlin/ru/missclick3/external/UserDTO.kt
package ru.missclick3.external

import kotlinx.serialization.Serializable

@Serializable
data class UserDTO(
    val id: String? = "",
    val username: String,
    val name: String,
    val role: String,
    val surname: String,
    val patronymic: String? = "",
    val email: String? = "",
    val phoneNumber: String? = "",
    val tgUsername: String? = "",
    val address: String,
    val password: String? = "",
    val salt: String? = ""
)
